charts/enterprise/traefik/values.yaml

```diff
@@ -414,3 +414,6 @@ persistence:
 portal:
   open:
     enabled: true
+    path: /dashboard/
+    override:
+      protocol: http
```

Comment:
you don't want the override here afaik.
As it would be https when using ingress and http when using LB

Reply:
Pushed a temp fix for that, until portal on library is adapted to detect those